app/models/penalties/latePayment/LPPPenaltyCategoryEnum.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.penalties.latePayment
+
+import play.api.libs.json._
+
+object LPPPenaltyCategoryEnum extends Enumeration {
+
+  val FirstPenalty: LPPPenaltyCategoryEnum.Value = Value("LPP1")
```

Comment:
same here, if we can use sealed trait along with standard patterns matching we can catch missing cases during  compile time